packages/api/package.json

```diff
@@ -14,6 +14,7 @@
     "test-coverage": "nyc npm test",
     "build": "webpack",
     "watch": "webpack --progress -w",
+    "build-lambda-zips": "for x in executeMigrations cleanExecutions app BulkDelete emsDistributionReport emsIngestReport emsProductMetadataReport indexer dbIndexer indexFromDatabase payloadLogger createReconciliationReport bootstrap; do (cd dist/${x} && rm -f lambda.zip && zip lambda.zip index.js); done",
```

Comment:
Is there a reason we didn't just do something like 

`for x in $(basename ./dist/*)?`

Reply:
We are now.